hpqc v0.0.31 nike/hybrid fails on power9

```
../../go/pkg/mod/github.com/katzenpost/hpqc@v0.0.31/nike/hybrid/no_nobs_csidh.go:11:40: undefined: scheme                                                     ../../go/pkg/mod/github.com/katzenpost/hpqc@v0.0.31/nike/hybrid/no_nobs_csidh.go:13:15: undefined: ecdh.NewEcdhNike 
```

--- a/nike/hybrid/no_nobs_csidh.go
+++ b/nike/hybrid/no_nobs_csidh.go
@@ -8,8 +8,8 @@ import (
 	"github.com/katzenpost/hpqc/rand"
 )
 
-var NOBS_CSIDH512X25519 nike.Scheme = &scheme{
+var NOBS_CSIDH512X25519 nike.Scheme = &Scheme{
 	name:   "NOBS_CSIDH-X25519",
-	first:  ecdh.NewEcdhNike(rand.Reader),
+	first:  ecdh.Scheme(rand.Reader),
 	second: nil,
 }

